packages/styles/scss/components/participant/_audio-visualizer.scss

```diff
@@ -1,11 +1,80 @@
+/**
+deprecated
+*/
 .audio-visualizer {
+  width: 100%;
+  height: 100%;
+  min-height: 160px;
+  background: var(--bg-control);
+  aspect-ratio: 16 / 9;
+
+  border-radius: 0.5rem;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  // gap: 0.5rem;
+
   & > rect {
-    fill: var(--bg2);
-    rx: 0.5rem;
+    fill: var(--accent-bg);
+    // box-shadow: 2px 2px 1px rgba(255, 255, 255, 0.4);
+    // transition: transform 40ms cubic-bezier(0.42, -0.39, 0.61, 1.4);
+    transition: transform 100ms cubic-bezier(0.19, 0.02, 0.09, 1);
   }
-  & > g > rect {
-    fill: var(--fg);
-    transition: height 100ms ease-out, y 100ms ease-out;
-    rx: 0.25rem;
+
+  & > path {
+    stroke: var(--accent-bg);
+    // box-shadow: 2px 2px 1px rgba(255, 255, 255, 0.4);
+    // transition: transform 40ms cubic-bezier(0.42, -0.39, 0.61, 1.4);
+    transition: 100ms cubic-bezier(0.19, 0.02, 0.09, 1);
   }
 }
+
+.audio-bar-visualizer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+  background: var(--bg);
+  gap: var(--va-bar-gap, 24px);
+
+  & > .audio-bar {
+    transform-origin: 'center';
+    height: 100%;
+    width: var(--va-bar-width, 12px);
+    border-radius: var(--va-bar-border-radius, 32px);
+    background-color: rgba(255, 255, 255, 0.05);
```

Comment:
@lukasIO maybe i'm just missing something (still new to react + tailwind etc) but how do i customize the bar color? I want it in gray/black on white. Is the only option to write a css rule for `.lk-audio-bar { background-color }`?

Reply:
ah, yeah, the un-highlighted bar colour is not exposed as a css variable right now. 
for hightlighted states you can use `--lk-fg`. Will add another one for the non-highlighted color state